Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build stage
+FROM python:3.10-slim AS build
+
+# Set work directory
+WORKDIR /app
+
+# Copy and install dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy application files
+COPY . .
+
+# Run stage
+FROM python:3.10-slim AS run
+
+# Set environment variables
+ENV PYTHONUNBUFFERED=1
+
+# Set work directory
+WORKDIR /app
+
+# Copy installed dependencies from the build stage
+COPY --from=build /usr/local/lib/python3.10/site-packages /usr/local/lib/python3.10/site-packages
```

Comment:
The COPY command for dependencies should target the specific directory where Python packages are installed. Ensure that the path is correct and that all necessary dependencies are included.